vcsh

```diff
@@ -279,7 +280,7 @@ list() {
 
 get_files() {
 	GIT_DIR=$VCSH_REPO_D/$VCSH_REPO_NAME.git; export GIT_DIR
-	git ls-files --full-name
+	git ls-files --full-name "$@"
```

Comment:
```suggestion
	git ls-files --full-name -- "$@"
```
Since we're handling user input here it's probably a good idea to guard against files that start with hyphens.